pennylane/__init__.py

```diff
@@ -82,13 +82,14 @@
 from pennylane.ops.functions import *
 from pennylane.optimize import *
 from pennylane.vqe import ExpvalCost, VQECost
-from pennylane.debugging import snapshots
+from pennylane.debugging.snapshots import snapshots
 
 # QueuingContext and collections needs to be imported after all other pennylane imports
 from .collections import QNodeCollection, dot, map, sum
 import pennylane.grouping  # pylint:disable=wrong-import-order
 import pennylane.gradients  # pylint:disable=wrong-import-order
 from pennylane.interfaces import execute  # pylint:disable=wrong-import-order
+import pennylane.debugging  # pylint:disable=wrong-import-order
```

Comment:
it might be good to make the `debugging` package accessible in the API section of the docs :)